SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -57,14 +58,14 @@ function Invoke-lpass {
     
     if ($lpassCommand -ne '' -and ((& "$lpassCommand" $lpassPath --version ) -like 'LastPass CLI*') ) {
         if ($InputObject) {
-            return $InputObject | & "$lpassCommand" $lpassPath @Arguments 
+            return $InputObject | & "$lpassCommand" $lpassPath @Arguments *>&1
```

Comment:
@TylerLeonhardt  By redirecting everything, it also kill that prompt. 

![image](https://user-images.githubusercontent.com/1980296/98455026-5eaa3e80-2139-11eb-9792-ab68e080f3ca.png)

More than that, there's no message to indicate that I need to login and instead it return an empty secret

![image](https://user-images.githubusercontent.com/1980296/98455044-b052c900-2139-11eb-8c1f-4fccf9a0b96b.png)

Valid or invalid secrets, everything is now an empty string. 

I encountered this initially when working on the previous PR and that's why I did settle down on doing it only on the `lpass show` within Set-Secret rather than everywhere.

Reply:
Hmmm what if we just redirect the error stream instead of _all_ streams?

Reply:
@TylerLeonhardt This seems to come from the error stream. `2>&1` do not produce the prompt either. 😞

Reply:
@TylerLeonhardt Not sure what happened with "past me" but I just tested again  and `2>&1` did work.. ¯\_(ツ)_/¯

Reply:
I submitted another PR to this branch with the suggested changes.